wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/ProtoFile.kt

```diff
@@ -129,24 +129,40 @@ data class ProtoFile(
     return result
   }
 
+  private fun referencedTypes(schema: Schema): List<Type> {
```

Comment:
I think this should return `ProtoType` objects, not `Type` objects. Ie. the call to `mapNotNull()` should be the caller’s responsibility.

(Just to make this more general purpose)

Reply:
I went with `Type` for consistency with `typesAndNestedTypes`, but happy to change to `ProtoType` if that’s preferred?

Reply:
Done.

Reply:
Thanks.

typesAndNestedTypes() doesn’t need to look things up in a schema because the types are defined within the file.

This thing doesn’t have that information so doing the schema lookup is extra work.